src/server.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const app = express();
+const cors = require("cors");
+require("dotenv").config({ path: "./config.env" });
```

Comment:
Is there a reason you're changing the path to the env file?

Reply:
Would `require('dotenv').config();` be correct ?

Reply:
It depends, obviously I can't tell if there's a .env file, or if there's a config.env file that's ignored, so I'm a little unclear why this is different to how we've demonstrated things.

Reply:
I'll have to ask @Minakrv after lunch because I don't have access to her .env file.